test/advanced.test.js

```diff
@@ -277,6 +287,21 @@ describe("mithril-objectify", function() {
                 '({tag:"div",attrs:undefined,children:[JSON.stringify({})],dom:undefined,domSize:undefined,events:undefined,key:undefined,state:{},text:undefined});'
             );
         });
+
+        it("should not convert JSON when entries are not objects", function() {
+            assert.equal(
+                code('m("div", JSON.stringify("string"))'),
+                'm("div",JSON.stringify("string"));'
+            );
+        });
+
+        it("should not convert JSON when entries are not literals", function() {
```

Comment:
Why is `JSON.stringify(identifier)` not safe to optimize? It always returns a string, and mithril will treat the returned string as a child text node.